cmd/stack_package.go

```diff
@@ -548,6 +548,13 @@ func CreateTemplateMap(labels map[string]string, stackYaml StackYaml, imageNames
 // previously created templateMetada to all files in the target directory
 func ApplyTemplating(stackPath string, templateMetadata interface{}) error {
 
+	// current method means we allow certain application file types through
+	allowedApplicationFiles := []string{
+		"application/json",
+		"application/javascript",
+		"application/x-python",
```

Comment:
FYI: https://stackoverflow.com/questions/45614485/what-is-the-mime-type-for-py-files

Reply:
mime type on this package (the one we are using) is application/x-python

https://github.com/gabriel-vasile/mimetype/blob/master/supported_mimes.md